src/services/alert/MessageProvider/status/index.js

```diff
@@ -58,9 +58,10 @@ module.exports = {
 			const rawData = await statusApi.getIncidents();
 			const instanceSpecific = [];
 			const noneSpecific = [];
+			const displayIncidentsInMin = Configuration.get('DISPLAY_INCIDENTS_IN_MIN');
 
 			const promises = rawData.data.map(async (element) => {
-				if (Date.parse(element.updated_at) + 1000 * 60 * 60 * 24 * 2 >= Date.now()) {
+				if (element.status !== 4 || Date.parse(element.updated_at) + displayIncidentsInMin >= Date.now()) {
```

Comment:
"The display of status events should not be removed until the event is set to "Behoben"." the or sound like this point is not solved with this condition.

Reply:
what does 4 mean??? 
can that be placed in a list somewhere above, and then in code use that var name instead

Reply:
The incident that is happening for more than 48 h should displayed
the resolved incident should be displayed for 48 hours (logic already exists).

Reply:
@virgilchiriac i already ask this in the last PR. The 4 should be the resolve status, that perfect fit the case.